test/modules/TestResultCoordinator/TestReportUtil.cs

```diff
@@ -156,7 +156,7 @@ internal static async Task UploadLogsAsync(string iotHubConnectionString, Uri bl
             ServiceClient serviceClient = ServiceClient.CreateFromConnectionString(iotHubConnectionString);
             CloudToDeviceMethod uploadLogRequest =
                 new CloudToDeviceMethod("UploadModuleLogs")
-                    .SetPayloadJson($"{{ \"schemaVersion\": \"1.0\", \"sasUrl\": \"{blobContainerWriteUri.AbsoluteUri}\", \"items\": [{{ \"id\": \".*\", \"filter\": {{}} }}], \"encoding\": \"gzip\" }}");
+                    .SetPayloadJson($"{{ \"schemaVersion\": \"1.0\", \"sasUrl\": \"{blobContainerWriteUri.AbsoluteUri}\", \"items\": [{{ \"id\": \".*\", \"filter\": {{}} }}], \"encoding\": \"none\",\"content-type\": \"text\" }}");
```

Comment:
so... we compressed each log and upload each log as a text stream?! I think we could compress it even more? (ps. I'm not sure how much memory the 2nd compression will help in term of size).

Reply:
The reason I went with text stream here is because the compression we were using previously was broken. Now we can at least see the logs.

Reply:
Before it was uploading a .gz that was not in a readable format.

Reply:
I also have separate bug logged to find out if this log upload is a product issue